client/client.go

```diff
@@ -324,6 +324,8 @@ const (
 	ErrorKindSnapAlreadyInstalled = "snap-already-installed"
 	ErrorKindSnapNotInstalled     = "snap-not-installed"
 	ErrorKindNoUpdateAvailable    = "snap-no-update-available"
+
+	ErrorKindNotSnap = "snap-not-snap"
```

Comment:
I would use "snap-not-a-snap"  as the tag, and maybe even the kind but no strong opinion on the kind